### PR TITLE
Fixes division by zero when 3d body does not have valid shape

### LINKED ISSUE #46738: PhysicsServer error with rigid BodySetSpace() GodotPhysics
**Godot version:**

3.2.3 mono official stable.

**OS/device including version:**

Windows 10 64 bit 

**Issue description:**

When setting the space of a rigid body from a PhysicsServer a stream of a repetitive error occurs with GodotPhysics enabled:
`ERROR: The axis Vector3 must be normalized.
   At: core/math/basis.cpp:980`

**Steps to reproduce in:**
1. Create a space
2. Set space to active
3. Create a body
4. Set body space to space we created earlier.
5. Enable GodotPhysics
6. Watch "The axis Vector3 must be normalized" spam.

**Minimal reproduction project:**

[godotBug.zip](https://github.com/godotengine/godot/files/6096110/godotBug.zip)

### PR DESCRIPTION
**Pull request by likeich**
Fixes #46738 by setting the default inertia to a valid value when there are no valid shapes for a 3d body.

Test Project: [Intertia Bug.zip](https://github.com/godotengine/godot/files/6104173/Intertia.Bug.zip)

### CODE REVIEW

File: servers/physics_3d/body_3d_sw.cpp [hunk 1]
```diff
@@ -102,6 +105,11 @@ void Body3DSW::update_inertias() {
 				inertia_tensor += shape_inertia_tensor + (Basis() * shape_origin.dot(shape_origin) - shape_origin.outer(shape_origin)) * mass;
 			}
 
+			// Set the inertia to a valid value when there are no valid shapes
```
Comment (pouleyKetchoupp) [inline servers/physics_3d/body_3d_sw.cpp:108]: The comment should end with a period to follow the style guide.
(lots of comments don't respect this style guide in the physics servers but it's better to do it on new code anyway)

**Review by pouleyKetchoupp — CHANGES_REQUESTED** [review]
Just a little detail about comment style, otherwise it looks good!

**Comment by pouleyKetchoupp** [comment]
This will require a separate PR for 3.2 since the cpp file was renamed on master.

File: servers/physics_3d/body_3d_sw.cpp (hunk 1 above)
Comment (likeich) [reply servers/physics_3d/body_3d_sw.cpp:108]: Got it. I'll change a few of the other comments in the file while I'm at it.

**Comment by likeich** [comment]
> This will require a separate PR for 3.2 since the cpp file was renamed on master.

I'll create that PR in a bit then as well.

**Review by pouleyKetchoupp — APPROVED** [review]

**Comment by pouleyKetchoupp** [comment]
PR on 3.2 branch: #46809

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**